README.md

```diff
@@ -764,6 +766,51 @@ It becomes then possible to retrieve past values :
 - ```<name[0]>``` uses the first value taken by the key
 - ```<name[1]>``` uses the second element taken by the key
 
+## JSON matchers
+
+The built-in JSON DSL makes it possible to express detailed assertions regarding the values of JSON fields.
+
+If a field is unstable you can also decide to ignore it completely but what if you do not care about the precise value but still want to make sure it has a certain property.
+
+JSON matchers are designed for this use case, they work more or less like placeholders in practice.
+
+```scala
+And assert body.ignoring("city", "realName", "publisher.location").is(
+  """
+  {
+    "name": "<<any-alphanum-string>>",
```

Comment:
The problem is that matcher input is always a string 
> There is a caveat though, unlike placeholders, the JSON matchers must be surrounded by quotes in all circumstances even if they are replacing JSON numbers or boolean.

so `<<any-string>>` would also match `1` or `true` so it's the same as `<<is-present>>` :(